docs/antora.yml

```
@@ -6,6 +6,7 @@ nav:
   - modules/ROOT/partials/nav.adoc
 asciidoc:
   attributes:
+    global: APPUiO Global
```

```suggestion
    global: APPUiO Cloud Global
```

APPUiO is a bit too unspecific, it could also be APPUiO Managed